src/oss/confluent.sh

```diff
@@ -1497,9 +1497,17 @@ produce_command() {
       export_kafka
       bootstrapserver="--broker-list localhost:${kafka_port}"
     fi
-    removestr="--value-format avro"
-    if [[ "$args" =~ "$removestr" ]]; then
-      args=${args//$removestr/}
+    cloud="--cloud"
+    if [[ "$args" =~ "$cloud" ]]; then
+      file="$HOME/.ccloud/config"
+      args=${args//$cloud/--producer.config $file}
+      BOOTSTRAP_SERVERS=$( grep "^bootstrap.server" $file | awk -F'=' '{print $2;}' )
+      BOOTSTRAP_SERVERS=${BOOTSTRAP_SERVERS/\\/}
```

Comment:
If this is needed because of ccloud config format we'd need a comment here.

Reply:
Agree